packages/kit/src/runtime/server/page/actions.js

```diff
@@ -280,6 +280,14 @@ function try_deserialize(data, fn, route_id) {
 		// If we're here, the data could not be serialized with devalue
 		const error = /** @type {any} */ (e);
 
+		// if someone tries to use `json()` in their action
+		if (data instanceof Response) {
```

Comment:
there's nothing really specific in this error message about a `Response`, so I'm wondering why the `if` below doesn't catch it. It seems like we could combine the two. Maybe they need to check if it's an instance of DevalueError or if name is DevalueError?

Reply:
It still catches it but the error message is usually "Cannot stringify arbitrary non-POJOs".